inst/builtin/bs5/shiny/_variables.scss

```diff
@@ -108,8 +108,13 @@ $box-shadow-dark:
   0px 3px 7px rgba(0, 0, 0,.1),
   0px 12px 30px rgba(0, 0, 0,.08) !default;
 
+// Cards
 $card-border-radius: 8px !default;
 
+// Borders
+$border-color-translucent: RGBA(0, 0, 0, 0.1) !default;
```

Comment:
If you're open to a more detailed $gray-200 comp in RGBA, I think it'd be `RGBA(40, 70, 94, 0.1)`:

![colors](https://github.com/rstudio/bslib/assets/5993637/69ce2351-2e28-402b-a986-621b409fbc3c)

Reply:
I'm open to that! We'd need a similar color for dark mode, too